### PR TITLE
MAINT(vcpkg): Add "qt5-translations", remove "opus"

### PR DESCRIPTION
**Pull request by davidebeatrici**
`qt5-translations` provides translations for Qt strings, such as the text of the standard button "OK".

`opus` was removed in 6b7897c7e5c95ae728af3f092121f9cc685ab3ed, but only for the PowerShell script.

### CODE REVIEW

**Comment by davidebeatrici** [comment]
@Krzmbrzl Should the second commit perhaps have `FEAT` instead of `MAINT`?

File: scripts/vcpkg/get_mumble_dependencies.ps1 [hunk 1]
```diff
@@ -9,6 +9,7 @@ $vcpkgdir = $profiledir + "\vcpkg"
 $mumble_deps = "qt5-base",
                "qt5-svg",
                "qt5-tools",
+               "qt5-translations",
```
Comment (Krzmbrzl) [inline scripts/vcpkg/get_mumble_dependencies.ps1:12]: Different indentation

File: scripts/vcpkg/get_mumble_dependencies.sh [hunk 2]
```diff
@@ -44,9 +44,9 @@ VCPKGDIR=~/vcpkg
 mumble_deps='qt5-base,
             qt5-svg,
             qt5-tools,
+            qt5-translations,
```
Comment (Krzmbrzl) [inline scripts/vcpkg/get_mumble_dependencies.sh:47]: Different indentation

**Review by Krzmbrzl — COMMENTED** [review]
Both commits should be `MAINT` imo.
This is because refactoring refers to production  code being changed without changing the functionality.

The `FEAT` part for the translations is my commit that bundles the translations into Mumble :shrug:

File: scripts/vcpkg/get_mumble_dependencies.ps1 (hunk 1 above)
Comment (davidebeatrici) [reply scripts/vcpkg/get_mumble_dependencies.ps1:12]: Are you sure? They all appear as spaces to me.

File: scripts/vcpkg/get_mumble_dependencies.sh (hunk 2 above)
Comment (davidebeatrici) [reply scripts/vcpkg/get_mumble_dependencies.sh:47]: Same as above.

File: scripts/vcpkg/get_mumble_dependencies.ps1 (hunk 1 above)
Comment (Krzmbrzl) [reply scripts/vcpkg/get_mumble_dependencies.ps1:12]: Oh lol - The GitHub app is trash. It displayed a different indentation for me xD

Comment (davidebeatrici) [reply scripts/vcpkg/get_mumble_dependencies.ps1:12]: Oh, wow...

**Review by Krzmbrzl — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
Note: Wait with merging though as I want to merge the translations PR first in order to resolve the conflicts on Weblate :point_up:

**Comment by davidebeatrici** [comment]
Feel free to merge.

**Pull request merged**